### PR TITLE
Enable use of sidetags for downstream automation

### PR DESCRIPTION
**Pull request by nforro**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -187,17 +187,24 @@ jobs:
...
 
   - job: bodhi_update
     trigger: commit
-    packages: [specfile]
+    packages: [specfile-epel8]
     packit_instances: ["stg"]
     allowed_builders: ["packit-stg", "packit"]
     dist_git_branches:
-      - fedora-branched # rawhide updates are created automatically
-      - epel-9
```
Comment (majamassarini) [inline .packit.yaml:202]: This is just temporary, right? Because we are missing the bodhi updates for the side tags?

**Review by majamassarini — APPROVED** [review]

File: .packit.yaml (hunk 1 above)
Comment (nforro) [reply .packit.yaml:202]: No, this is Bodhi update for `specfile-epel8` package, neither ogr nor packit support EPEL8 anymore.

Comment (majamassarini) [reply .packit.yaml:202]: now I see the bodhi job in the packit package...

Comment (nforro) [reply .packit.yaml:202]: `bodhi_update` job for ogr, specfile and packit is configured only in packit.

**Pull request merged**